dear_petition/petition/tests/test_models.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+
+from .factories import BatchFactory, record_data
+
+from ..models import CIPRSRecord
+
+from ..utils import dt_obj_to_date
+
+pytestmark = pytest.mark.django_db
+
+
+def test_ciprs_record_create():
```

Comment:
Can we also test more uses of this method, like:
 - pass an empty `data` dictionary
 - call `create_record()` for the same `data` multiple times
 - call `create_record()` for the same `batch`, `label, and `data` multiple times
?

Reply:
done